force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -52,6 +89,14 @@ public class CollectionsTasks {
     List<String> codeTicks,
     List<Integer> priceTicks
   ) {
-    return 0;
+
```

Comment:
Imagine the input size is 100 000.
The stock price we are interested in is met only couple of times in the array.

How we can make the algorithm more efficient?